core/bin/external_node/src/main.rs

```diff
@@ -55,6 +60,42 @@ mod version_sync_task;
 
 const RELEASE_MANIFEST: &str = include_str!("../../../../.github/release-please/manifest.json");
 
+async fn sync_state_updater(
+    sync_state: SyncState,
+    tree_pool: ConnectionPool<Core>,
+    main_node_client: HttpClient,
+    stop_receiver: watch::Receiver<bool>,
+) -> anyhow::Result<()> {
+    const UPDATE_INTERVAL: Duration = Duration::from_secs(60);
```

Comment:
isn't 60 seconds too much? we consider node out of sync after I think being t10 blocks behind, which takes 10 seconds on average. So I'd set 10 seconds interval

Reply:
```suggestion
    const UPDATE_INTERVAL: Duration = Duration::from_secs(10);
```